x/fungible/keeper/deposits_test.go

```diff
@@ -352,3 +353,17 @@ func TestKeeper_ZRC20DepositAndCallContract(t *testing.T) {
 		require.EqualValues(t, int64(0), balance.Int64())
 	})
 }
+
+func TestKeeper_DepositCoinZeta(t *testing.T) {
+	k, ctx, _, _ := testkeeper.FungibleKeeper(t)
+	to := sample.EthAddress()
+	amount := big.NewInt(1)
+	zetaToAddress := sdk.AccAddress(to.Bytes())
+
+	b := k.GetBankKeeper().GetBalance(ctx, zetaToAddress, "azeta")
+	require.Equal(t, int64(0), b.Amount.Int64())
+
+	k.DepositCoinZeta(ctx, to, amount)
+	b = k.GetBankKeeper().GetBalance(ctx, zetaToAddress, "azeta")
```

Comment:
Can't we use sdk keepers and not have to add GetBalance to the expected keepers?